- hosts: localhost
  gather_facts: False
  connection: local
  - task:
      name: "Valida existencia de bucket." 
        debug: 
          msg: "El bucket {{ bucket_name }} no existe."
